src/printer/layers.js

```diff
@@ -232,3 +238,34 @@ function createLayerWMS(layerSpec, rootFrameState) {
 
   return progress$;
 }
+
+/**
+ * @param {WmtsLayer} layerSpec
+ * @param {FrameState} rootFrameState
+ * @return {Observable<LayerPrintStatus>}
+ */
+function createLayerWMTS(layerSpec, rootFrameState) {
+  let { matrixSet, tileGrid, projection } = layerSpec;
+  let { resolutions, extent, matrixIds } = tileGrid;
+  projection = projection || getProjection(matrixSet);
```

Comment:
Some explanation about this would be nice in the README. Do we really expect the `matrixSet` to bear the name of a projection if no projection is provided ?

Reply:
You are right I did that in the beginning because of the WMTS example I used but it make no sense IMO.